cmake/projects/OpenSSL/schemes/url_sha1_openssl.cmake.in

```diff
@@ -52,7 +52,13 @@ else()
   set(configure_command "./config")
 endif()
 
-hunter_dump_cmake_flags(SKIP_INCLUDES)
+set(skip_pic "")
+if(APPLE)
+  # Apple cannot process `-fPIC` flag argument (PR-1121)
+  set(skip_pic SKIP_PIC)
+endif()
+
+hunter_dump_cmake_flags(SKIP_INCLUDES ${skip_pic})
```

Comment:
`hunter_dump_cmake_flags` doesn't support `SKIP_PIC` currently, you have to add this functionality.

Reply:
OK got it.